From 27c7bc970d9e7ffe060688a8dd77b7747503a564 Mon Sep 17 00:00:00 2001
From: Pat Riehecky <jcpunk@gmail.com>
Date: Fri, 9 Jun 2017 17:48:25 -0500
Subject: [PATCH] man: systemd-timesyncd.service(8) (#6109)

Updates the documentation to note use of SNTP (resolves #5735)
(cherry picked from commit 42d3bf86bb75842602d3712caa2baccd09a1c795)
---
 man/systemd-timesyncd.service.xml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/man/systemd-timesyncd.service.xml b/man/systemd-timesyncd.service.xml
index 6ec384313b..3edcaf1b4e 100644
--- a/man/systemd-timesyncd.service.xml
+++ b/man/systemd-timesyncd.service.xml
@@ -64,6 +64,12 @@
     reboots to ensure it monotonically advances even if the system
     lacks a battery-buffered RTC chip.</para>
 
+    <para>The <filename>systemd-timesyncd</filename> service
+    specifically implements only SNTP. This minimalistic
+    service will set the system clock for large offsets or
+    slowly adjust it for smaller deltas. More complex use
+    cases are not covered by <filename>systemd-timesyncd</filename>.</para>
+
     <para>The NTP servers contacted are determined from the global
     settings in
     <citerefentry><refentrytitle>timesyncd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
